src/IoTHub.Portal.Tests.Unit/Infrastructure/Services/LoRaWanManagementServiceTests.cs

```diff
@@ -18,6 +18,8 @@ namespace IoTHub.Portal.Tests.Unit.Infrastructure.Services
     using NUnit.Framework;
     using RichardSzalay.MockHttp;
     using UnitTests.Bases;
+    using System.Linq;
+    using Fare;
 
     [TestFixture]
     public class LoRaWanManagementServiceTests : BackendUnitTest
```

Comment:
@lucas-mrq can you add a hardcoded test with the hex Frame to send and the corresponding base64 encoded frame verification?

Reply:
I did it in the last commit